aptos-move/framework/aptos-framework/sources/vesting.move

```diff
@@ -1460,6 +1475,84 @@ module aptos_framework::vesting {
         assert!(operator_commission_percentage(contract_address) == 10, 0);
     }
 
+    #[test(aptos_framework = @0x1, admin = @0x123, operator = @0x345)]
+    public entry fun test_update_commission_percentage(
```

Comment:
This test seems redundant with the other test added below. Does this test any code path that the other test doesn't?

Reply:
It is a more basic test case, but it seems redundant. It's merged into the other one now.